pkgs/applications/radio/openwebrx/default.nix

```diff
@@ -0,0 +1,92 @@
+{ stdenv, lib, buildPythonPackage, buildPythonApplication, fetchFromGitHub
+, pkg-config, cmake, setuptools
+, rtl-sdr, soapysdr-with-plugins, csdr, direwolf
+}:
+
+let
+
+  js8py = buildPythonPackage rec {
```

Comment:
Is there a reason that this package is just defined in a "let"? This could be a separate package. Where is the `JS8Call` binary supposed to come from?

Reply:
The package is developed by the same developer and is only used in openwebrx. I am fine with this being in a let in and extra.